packages/kerosene-ui/src/hooks/useRect.spec.tsx

```diff
@@ -92,7 +92,7 @@ describe("useRect", () => {
 
   it("should add and remove listeners for event specified", () => {
     const Component = () => {
-      const [ref] = useRect(false, "transitionend");
+      const [ref] = useRect(false, ["transitionend"]);
```

Comment:
Do you think it's worth re-adding the check for the scroll listener for this case?